db/articles.js

```diff
@@ -0,0 +1,75 @@
+//jshint esversion:6
+class Articles {
+  constructor(){
+    this._collection = [];
+  }
+
+  getArticles (){
+    return this._collection;
+  }
+  post (article){
+    let isRepost = articleIsRepost(article.title, this._collection);
```

Comment:
I'm not sure if I'm getting this right but are you only adding a new article if an article with the same title already exists?